lib/splitclient-rb/clients/split_client.rb

```diff
@@ -1,4 +1,6 @@
 module SplitIoClient
+  PROPERTIES_SIZE_THRESHOLD = 32768
```

Comment:
32kb is the maximum event size. not payload size. We consider each event to have a starting point of 1024 bytes, based on a worst case avg, properties + 1024 should be less than 5mb

Reply:
Changed to `EVENTS_SIZE_THRESHOLD`